src/components/FieldOverviewCard/FieldOverviewCard.tsx

```diff
@@ -0,0 +1,80 @@
+import { Card, makeStyles, SvgIconProps, Typography } from '@material-ui/core';
+import { DateTime } from 'luxon';
+import { FC } from 'react';
+import * as React from 'react';
+import { ButtonLink, CardActionAreaLink } from '../Routing';
+
+const useStyles = makeStyles(({ spacing, palette }) => ({
+  card: {
+    width: '321px',
+  },
+  topArea: {
+    display: 'flex',
+    padding: spacing(3, 4, 3, 4),
+    borderBottom: `0.5px solid ${palette.grey[300]}`,
+  },
+  icon: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: '64px',
+    height: '64px',
+    backgroundColor: palette.grey[200],
+    borderRadius: '64px',
+  },
+  rightContent: {
+    paddingLeft: spacing(4),
+  },
+  bottomArea: {
+    display: 'flex',
+    padding: spacing(1, 2, 1, 1),
+    alignItems: 'center',
+    justifyContent: 'space-between',
+  },
+}));
+
+export interface FieldOverviewCardProps {
+  title: string;
+  value: string;
+  icon: React.ComponentType<SvgIconProps>;
+  updatedAt: DateTime;
+  to: string;
+  viewLabel: string;
+}
+
+export const FieldOverviewCard: FC<FieldOverviewCardProps> = ({
+  title,
+  value,
+  icon: Icon,
+  updatedAt,
+  to,
+  viewLabel,
+}) => {
+  const classes = useStyles();
+
+  return (
+    <Card className={classes.card}>
+      <CardActionAreaLink to={to} className={classes.topArea}>
+        <div className={classes.icon}>
+          <Icon />
```

Comment:
We should try to get the blue here. Either css or maybe `htmlColor` prop.

Also, `fontSize="large"`.

Also, maybe we should move this to separate comp. We will already need to use it in another spot. `HugeIcon` maybe? lol

I'm also wondering if we should to to change it to a ReactNode and let them pass in the created element. We would probably need to clone it, but that would allow tweaking.
For example, the CORD icon looks better offset 4px to the left because human eyes 🤪